libsolidity/ast/TypeProvider.h

```diff
@@ -139,6 +139,8 @@ class TypeProvider
 	/// @returns the function type of an event.
 	static FunctionType const* function(EventDefinition const& _event);
 
+	static FunctionType const* function(ErrorDefinition const& _error);
```

Comment:
All the other functions here have docstrings so this one should too.

```suggestion
	/// @returns the function type of an error.
	static FunctionType const* function(ErrorDefinition const& _error);
```

Reply:
True, but the docstrings are also no-ops.